packages/runtime/container-runtime/src/test/blobManager.spec.ts

```diff
@@ -310,6 +319,10 @@ describe("BlobManager", () => {
 	});
 
 	afterEach(async () => {
+		if (skipAfterEach) {
```

Comment:
i don't quite understand why we need this, can we make it work is some way?

Reply:
if it is supposed to work this way, please move to a new block, either a new describe or new file, and validate this value is what you expect.